mingw-w64-git/git-bash.rc

```diff
@@ -4,4 +4,4 @@ BEGIN
 	1 "SHOW_CONSOLE=1 APPEND_QUOTE=1 @@COMSPEC@@ /S /C """"@@EXEPATH@@\\usr\\bin\\bash.exe"" --login -i"
 END
 
-"MAINICON" ICON "git-for-windows.ico"
+0 ICON "git-for-windows.ico"
```

Comment:
This failed with `llvm-rc: Error parsing file: expected int or identifier, got "MAINICON"`, so I had to change this. Will need to check if that also still works on `mingw64`. If it does, would you be OK with updating the file like this? Or would you like me to create separate `.rc` files for `clangxxxxx` vs `mingwxx`?

Reply:
Hmm. According to https://learn.microsoft.com/en-us/windows/win32/menurc/icon-resource, it should work with a unique name (but maybe clang requires it to not be quoted?). In any case, could you verify that the generated `git-bash.exe` shows the intended icon, i.e. [this `.ico`](https://raw.githubusercontent.com/git-for-windows/MINGW-packages/2489eb218fef293b1b9c8f84241691401c910ef9/mingw-w64-git/git-for-windows.ico)?

Reply:
Oh, and before I forget: please move this into its own commit.